Jenkinsfile

```diff
@@ -438,7 +426,7 @@ pipeline {
                         curl -v -T $LOCAL_ARTIFACT2 -u$BINTRAY_CREDS_USR:$BINTRAY_CREDS_PSW $BINTRAY_LOCATION/$BINTRAY_ARTIFACT2?override=1
                         curl -v -T $LOCAL_JENKINSFILE -u$BINTRAY_CREDS_USR:$BINTRAY_CREDS_PSW $BINTRAY_LOCATION/$BINTRAY_JENKINSFILE?override=1
                         echo 'publish file in bintray'
-                        curl -v -X POST -u$BINTRAY_CREDS_USR:$BINTRAY_CREDS_PSW $BINTRAY_LOCATION/predix-uaa/publish
+                        curl -v -X POST -u$BINTRAY_CREDS_USR:$BINTRAY_CREDS_PSW $BINTRAY_LOCATION/publish
```

Comment:
Have you tried this command.  I believe to publish you don't need to specify app name; it doesn't work.  Publish command should be:

curl -v -X POST -u$BINTRAY_CREDS_USR:$BINTRAY_CREDS_PSW **https://api.bintray.com/content/gedigital/Rosneft/uaa/${APP_VERSION}/publish**